src/Functional.Primitives.FluentAssertions/Functional.Primitives.FluentAssertions/OptionTypeAssertions.cs

```diff
@@ -90,7 +49,20 @@ public void NotHaveValue(string because = "", params object[] becauseArgs)
 			Execute.Assertion
 				.ForCondition(!_subject.HasValue())
 				.BecauseOf(because, becauseArgs)
-				.FailWith($"Expected to not have value, but received a value instead {{reason}}");
+				.FailWith(FailReasonForNotHaveValue);
 		}
+
+		private FailReason FailReasonForNotHaveValue()
+		{
+			return new FailReason("Expected to not have value, but received a value instead{reason}:"
```

Comment:
These comments also apply to `FailReasonForBeSuccessful()` and `FailReasonForBeFaulted()` in `ResultTypeAssertions.cs`.

Reply:
`{reason}` is a placeholder string that is filled in my `FluentAssertions` with " because [whatever reason string the developer types in]", hence the lack of space.  I agree that the reason can be moved to the middle of the sentence.